pkg/ublox/types.go

```diff
@@ -0,0 +1,69 @@
+package ublox
+
+// // Get GNSS Antenna receiver antenna status in both blocks
+// # ubxtool -v 1 -P 29.20 -p MON-RF
+// UBX-MON-RF:
+// Poll request
+// UBX-MON-RF:
+// version 0 nBlocks 2 reserved1 0 0
+// blockId 0 flags x0 antStatus 2 antPower 1 postStatus 0 reserved2 0 0 0 0
+// blockId 1 flags x0 antStatus 2 antPower 1 postStatus 0 reserved2 0 0 0 0
+type ANT_STATUS int
+type POWER_STATUS int
+
+const (
+	OFF POWER_STATUS = iota
+	ON
+)
+const (
+	NOT_OK ANT_STATUS = iota
+	UNKNOW
+	OK
+)
+
+func (p POWER_STATUS) String() string {
+	return [...]string{"OFF", "ON"}[p]
+}
+func (p POWER_STATUS) IntString() string {
+	return [...]string{"0", "1"}[p]
+}
+
+func (a ANT_STATUS) String() string {
+	return [...]string{"NOT_OK", "UNKNOWN", "OK"}[a]
```

Comment:
These are different from Intel E810-XXVDA4T user guide, Section 4.15.2:
```
Status of the antenna supervisor state machine (0x00=INIT,0x01=DONTKNOW,
0x02=OK,0x03=SHORT,0x04=OPEN)
Current power status of antenna (0x00=OFF,0x01=ON,0x02=DONTKNOW)
```

Reply:
power status and Ant power are different thing.We are not using it now .